app/graphql/mutations/create_dynamic_field_category.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class Mutations::CreateDynamicFieldCategory < Mutations::BaseMutation
+  argument :display_label, String, required: true
+  argument :sort_order, Integer, required: true
```

Comment:
`sort_order` actually doesn't have to be required by this endpoint even though it is required in the database schema. There is logic in `DynamicFieldStructure::Sortable` that sets `sort_order` if one isn't present. It's kind of a hidden feature.

Reply:
The same applies to dynamic field group and dynamic field

Reply:
Related: Please see my comment about sort_order in `Mutations::CreateDynamicFieldGroup`.